packages/paste-website/src/pages/components/anchor/index.mdx

```diff
@@ -55,7 +60,7 @@ export const pageQuery = graphql`
 
 <contentwrapper>
 
-<TableOfContents headings={props.data.mdx.headings} />
+<PageAside data={props.data.mdx} headings={props.data.mdx.headings} />
```

Comment:
Could we technically pass just `data={props.data.mdx}` and pull headings from that in the `PageAside` component?

Reply:
Oh right. Good call. I've made that change.